Source/FaceFX/Private/Sequencer/FaceFXAnimationSectionTemplate.cpp

```diff
@@ -155,6 +156,7 @@ void FFaceFXAnimationSectionTemplate::TearDown(FPersistentEvaluationData& Persis
 			return;
```

Comment:
I found another problem while editing.

TrackData will not remove on sequence edit due to function returned by key id nullptr.

Reply:
It caused by lost reference of Actor.
I opened level which actor in and play again, the Key is not nullptr